x-pack/legacy/plugins/transform/public/app/sections/clone_transform/clone_transform_section.tsx

```diff
@@ -0,0 +1,194 @@
+    docTitleService.setTitle('createTransform');
+  }, []);
+
+  const api = useApi();
+
+  const transformId = match.params.transformId;
+
+  const [transformConfig, setTransformConfig] = useState<TransformPivotConfig>();
+  const [errorMessage, setErrorMessage] = useState();
+  const [isInitialized, setIsInitialized] = useState(false);
+  const [savedObjectId, setSavedObjectId] = useState<string | undefined>(undefined);
+
+  const fetchTransformConfig = async () => {
+    try {
+      const transformConfigs: GetTransformsResponse = await api.getTransforms(transformId);
+      if (isGetTransformsResponseError(transformConfigs)) {
+        setTransformConfig(undefined);
+        setErrorMessage(transformConfigs.error.msg);
+        setIsInitialized(true);
+        return;
+      }
+
+      await loadIndexPatterns(savedObjectsClient, indexPatterns);
+      const indexPatternTitle = Array.isArray(transformConfigs.transforms[0].source.index)
+        ? transformConfigs.transforms[0].source.index.join(',')
+        : transformConfigs.transforms[0].source.index;
+      const indexPatternId = getIndexPatternIdByTitle(indexPatternTitle);
+
+      if (indexPatternId === undefined) {
+        throw new Error(
+          i18n.translate('xpack.transform.clone.errorPromptText', {
+            defaultMessage: 'Could not fetch the Kibana index pattern ID.',
+          })
+        );
+      }
+
+      setSavedObjectId(indexPatternId);
+
+      setTransformConfig(transformConfigs.transforms[0]);
+      setErrorMessage(undefined);
+      setIsInitialized(true);
+    } catch (e) {
+      setTransformConfig(undefined);
+      if (e.message !== undefined) {
+        setErrorMessage(e.message);
+      } else {
+        setErrorMessage(JSON.stringify(e, null, 2));
+      }
+      setIsInitialized(true);
+    }
+  };
+
+  useEffect(() => {
+    fetchTransformConfig();
+    // The effect should only be called once.
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  return (
+    <PrivilegesWrapper privileges={APP_CREATE_TRANSFORM_CLUSTER_PRIVILEGES}>
+      <EuiPageContent data-test-subj="transformPageCloneTransform">
+        <EuiTitle size="l">
+          <EuiFlexGroup alignItems="center">
+            <EuiFlexItem grow={true}>
+              <h1>
+                <FormattedMessage
+                  id="xpack.transform.transformsWizard.cloneTransformTitle"
+                  defaultMessage="Clone transform"
+                />
+                <span>&nbsp;</span>
+                <EuiBetaBadge
+                  label={i18n.translate('xpack.transform.transformsWizard.betaBadgeLabel', {
+                    defaultMessage: `Beta`,
+                  })}
+                  tooltipContent={i18n.translate(
+                    'xpack.transform.transformsWizard.betaBadgeTooltipContent',
+                    {
+                      defaultMessage: `Transforms are a beta feature. We'd love to hear your feedback.`,
```

Comment:
shall we remove the beta tag for 7.7?

Reply:
yes, this will be done in a separate PR.